src/control/server/mgmt_svc.go

```diff
@@ -532,3 +532,27 @@ func (svc *mgmtSvc) ListPools(ctx context.Context, req *mgmtpb.ListPoolsReq) (*m
 
 	return resp, nil
 }
+
+// ListContainers implements the method defined for the Management Service.
+func (svc *mgmtSvc) ListContainers(ctx context.Context, req *mgmtpb.ListContReq) (*mgmtpb.ListContResp, error) {
```

Comment:
I'll try and let you know if I have issues. Generally have been focused on the C part of the implementation.

Reply:
Sure. When we add the dmg command we could add the tests at that time. That's what I ended up doing with ListPools. But I don't want it to be forgotten, so it would be best if you can address it now.